Traceback (most recent call last):
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mriki/.rye/py/cpython@3.11.8/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
class TanhNet(nn.Module):
  def __init__(self, hidden_size=100, n_class=3):
    super().__init__()
    self.feature_extractor = nn.Sequential(
          nn.Linear(4, hidden_size), # 1
          nn.Tanh(),
          nn.Linear(hidden_size, hidden_size), # 2
          nn.Tanh(),
          ... # 3
          ... # 4
          ... # 5
      )
    self.classifier = nn.Linear(hidden_size, n_class)

  def forward(self,x):
    x = self.feature_extractor(x)
    return self.classifier(x)
------------------


[0;36m  Cell [0;32mIn[19], line 9[0;36m[0m
[0;31m    ... # 3[0m
[0m    ^[0m
[0;31mSyntaxError[0m[0;31m:[0m invalid syntax. Perhaps you forgot a comma?


